<?xml version="1.0"?>
   <robot name="Roboo">
    <link name="base_link">
        <visual>
          <geometry>
            <box size="0.6 0.1 0.1"/>
          </geometry>
          <material name="orange">
    <color rgba="1.0 0.423529411765 0.0392156862745 1.0"/>
  </material>
        </visual>
      </link>
   <link name="right_wheel">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.1"/>
      </geometry>
      <origin rpy="1.57075 0 0" xyz="0 -0.02 0"/>
    </visual>
  </link>

  <link name="left_wheel">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.1"/>
      </geometry>
      <origin rpy="1.57075 0 0" xyz="0 0.02 0"/>
    </visual>
  </link>

  <joint name="base_to_right_wheel" type="revolute">
    <limit effort="1000.0" lower="-3.14" upper="3.14" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="right_wheel"/>
    <origin xyz="0.2 -0.05 0"/>
    <axis xyz="0 1 0"/>
  </joint>

  <joint name="base_to_left_wheel" type="revolute">
    <limit effort="1000.0" lower="-3.14" upper="3.14" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="left_wheel"/>
    <origin xyz="0.2 0.05 0"/>
    <axis xyz="0 1 0"/>
  </joint>



  <link name="right_wheel2">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.1"/>
      </geometry>
      <origin rpy="1.57075 0 0" xyz="0 -0.02 0"/>
    </visual>
  </link>

  <link name="left_wheel2">
    <visual>
      <geometry>
        <cylinder length="0.04" radius="0.1"/>
      </geometry>
      <origin rpy="1.57075 0 0" xyz="0 0.02 0"/>
    </visual>
  </link>

  <joint name="base_to_right_wheel2" type="revolute">
    <limit effort="1000.0" lower="-3.14" upper="3.14" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="right_wheel2"/>
    <origin xyz="-0.2 -0.05 0"/>
    <axis xyz="0 1 0"/>
  </joint>

  <joint name="base_to_left_wheel2" type="revolute">
    <limit effort="1000.0" lower="-3.14" upper="3.14" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="left_wheel2"/>
    <origin xyz="-0.2 0.05 0"/>
    <axis xyz="0 1 0"/>
  </joint>

   </robot>